src/languageFacts/colors.ts

```diff
@@ -190,10 +190,24 @@ function getNumericValue(node: nodes.Node, factor: number) {
 
 function getAngle(node: nodes.Node) {
 	const val = node.getText();
-	const m = val.match(/^([-+]?[0-9]*\.?[0-9]+)(deg)?$/);
+	const m = val.match(/^([-+]?[0-9]*\.?[0-9]+)(deg|rad|grad|turn)?$/);
 	if (m) {
-		return parseFloat(val) % 360;
+		switch (m[2]) {
+			case 'deg':
+				return parseFloat(val) % 360;
+			case 'rad':
+				return parseFloat(val) * 180 / Math.PI;
```

Comment:
For 'rad', 'grad' and 'turn', can they represent multiple turns? Should we add a modulo 360 as well?

Reply:
🤔 Will check.
Good catch!

Reply:
This has been updated.